### PR TITLE
Completed CSS Brochure Form

### PR DESCRIPTION
**Pull request by JoshuaAaronBaker**
Screen Shot 2022-06-30 at 9 56 46 AM

### CODE REVIEW

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Josh, your form is too narrow as compared to the example, please revise and submit a new screenshot with your revision.

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+div {
+  margin-top: 0.5rem;
+}
+
+::placeholder {
+  font-size: 10px;
+  font-family: AtlasGrotesk, sans-serif;
+  opacity: 0.7;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:80]: You shouldn't use opacity to change the color of text, use the `color` property

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by JoshuaAaronBaker** [comment]
Screen Shot 2022-06-30 at 12 52 28 PM

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Review by codingcodymiller — APPROVED** [review]
Nice work!

**Pull request merged**